network-api/networkapi/wagtailpages/templates/wagtailpages/tags/card-cta.html

```diff
@@ -0,0 +1,34 @@
+{% load wagtailcore_tags wagtailimages_tags %}
+
+<div class="row card-cta" style="background: #F5F5F5;">
+  <div class="col-12 col-md-6 p-5">
+    <h3 class="">{{ title }}</h3>
+    <p>{{ description | richtext }}</p>
+
+    {% if commitment %}
+    <div class="commitment mb-3">{{ commitment }}</div>
+    {% endif %}
+
+    {% if link_url %}
+    <a class="btn btn-ghost" href="{{ link_url }}">{{ link_label }}</a>
+    {% endif %}
+
+    <div class="social-links d-flex">
+      {% if facebook %}
+      <a class="social-link social-link-facebook" href="https://www.facebook.com/sharer/sharer.php?u={{ facebook }}"></a>
+      {% endif %}
+
+      {% if twitter %}
+      <a class="social-link social-link-twitter" href="https://twitter.com/home?status={{ twitter }}"></a>
+      {% endif %}
+
+      {% if email_body %}
```

Comment:
do we need to check `email_subject` here as well?

Reply:
Yeah, I guess that's a good idea.